### DISCUSSION TITLE
get_latest_vehicle_data() doesn't work

### DISCUSSION THREAD

**Discussion by micves** [tdorssers/TeslaPy#123] [Q&A] [answered]
I have other functions working, like these:
```
vehicles = tesla.vehicle_list()
vehicle = vehicles[0]
current_state = vehicle.get_vehicle_summary()['state']
print(str(datetime.now()) + ': ' + current_state)
```
But get_latest_vehicle_data() gives me 404 no matter what state the car is in (online or offline).
```
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://owner-api.teslamotors.com/api/1/vehicles/{id}/latest_vehicle_data
```
vehicle_data() works fine in online state.

I'm trying to figure out why my Model S 2017 with MCU2 is having weird sleep pattern (offline 1hour, online 3minutes).
It seems to be a common issue with MCU2 upgraded cars. 
Making the issue worse is for instance having teslamate running that polls the car (vehicle_data) when it comes online, which extends the online time.

I was hoping latest_vehicle_data could give data without extending the time.

Is anyone else having issues with latest_vehicle_data or is it just a MCU2 upgrade thing?

**Comment by israndy** [comment]
Here I thought it was me, I ended up replacing all the get_latest calls in my code. It was an exciting prospect when I learned it was an option, glad I am not the only one having issues

**Reply by tdorssers** [reply, marked as answer]
I have removed get_latest in the latest commit :) The call was removed from the endpoints JSON a while ago and now the endpoint no longer exists on the servers. It looks like get_vehicle_data is now similar to get_latest, as it returns cached data when the vehicle is asleep.
